tests/test_mf6_dis_bmi.py

```diff
@@ -319,6 +345,26 @@ def test_get_value_int_scalar(flopy_dis_idomain_mf6):
     )
 
 
+def test_get_value_bool(flopy_dis_idomain_mf6):
+    """Test get_value with bool (LOGICAL) type."""
+    mf6 = flopy_dis_idomain_mf6[1]
+    if mf6_version_tuple(mf6.get_version()) < (6, 5, 0):
```

Comment:
I assume you checked that comparing tuples gives the expected behavior? :)

Reply:
Yes, it works with versions higher and lower. This skip condition is needed since the tests only run on the modflow6 release, not develop version. It would be desirable to test modflow6 nightly repo, but this would take a bit of a refactor to do.

Reply:
>  It would be desirable to test modflow6 nightly repo, but this would take a bit of a refactor to do.

It's actually not too hard.
Only one line needs to be changed, as can be seen here: https://github.com/Deltares/xmipy/pull/98/files
Feel free to add this to this PR if you want to.

Reply:
See #108 which should have merge priority over this PR.